tests/queries/0_stateless/03228_async_insert_query_params.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# Tags: no-parallel
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+$CLICKHOUSE_CLIENT -q "
+    DROP TABLE IF EXISTS t_async_insert_params;
+    CREATE TABLE t_async_insert_params (id UInt64) ENGINE = Memory;
+"
+cmd_params="--async_insert 1 --async_insert_busy_timeout_max_ms 300000 --async_insert_busy_timeout_min_ms 300000 --wait_for_async_insert 0 --async_insert_use_adaptive_busy_timeout 0"
+
+$CLICKHOUSE_CLIENT $cmd_params -q "SET param_p1 = 11; INSERT INTO t_async_insert_params VALUES ({p1:UInt64});"
```

Comment:
Try to insert a value with an incompatible type?
https://pastila.nl/?046b4b6a/ebe22e9060d4186893456ddf7ee634e6#t45VkUbeNfojHDIapehSEw==